### ISSUE TITLE
Automatically run UI Tests after build

### ISSUE THREAD

**Issue by Carbaholic** [microsoft/appcenter#277]
**Describe the solution you'd like**
I would like to automatically run UI Tests on a given build

**Describe alternatives you've considered**
There is one guy that figured out how to run UI tests through hackery: 

https://tomsoderling.github.io/AppCenter-Automated-UI-tests-on-build/

**Additional context**
This would be very useful

**Comment by brminnick** [comment]
Here's a post-build script I wrote to build my Xamarin.UITest project and push it to App Center Test: 
https://github.com/brminnick/UITestSampleApp/blob/master/Src/UITestSampleApp.iOS/appcenter-post-build.sh

**Comment by craig95** [comment]
Yes, this would be very helpful. Not needing to create a custom post-build script.

**Comment by nrajpurkar** [comment]
Thanks for the feature request and to those adding their support. We'll continue to track this as a potential new feature. Thanks to @brminnick for your example as well!

**Comment by hemantbavle1** [comment]
How did you include the provisioning profiles? I keep getting the following error for provisioning profile? How does one resolve these errors? I spent hours on this..nothing worked!!! :/

`error: No profiles for 'com.xyzorg.appname' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.xyzorg.appnamer'. (in target 'TargetName')
`

`error: No profile for team 'ABCABC1234' matching 'DevProfileName' found: Xcode couldn't find any provisioning profiles matching 'ABCABC1234/DevProfileName'. Install the profile (by dragging and dropping it onto Xcode's dock item) or select a different one in the General tab of the target editor. (in target 'TestTargetUITests')`

**Comment by craig95** [comment]
@hemantbavle1 I used to use Bitrise.io in the past, they have a nice tool called [codesigndoc,](https://github.com/bitrise-io/codesigndoc) which gathers all the necessary profiles needed. I used it for AppCenter as well, just skipping the last step (uploading to Bitrise) and just uploaded the needed files generated to AppCenter manually. 
[Here](https://devcenter.bitrise.io/code-signing/ios-code-signing/collecting-files-with-codesigndoc/) is a link to their documentation for codesigndoc.
